### ISSUE TITLE
Font Pack concept / idea

### ISSUE THREAD

**Issue by jfrux** [Codeinwp/wp-menu-icons#19]
Hello,

Wanted to run this by you.
I have an idea for this plugin that would enhance it and allow it to be a bit more modular / expandable.

The idea is centered around font "packs" or "packages" that contain two folders... a font and css directory and possibly a config.json file with the exact same format as fontello.com uses for their package downloads.

Essentially, it would allow anyone to piece together there own icon font and drag it into the fontpacks directory and your plugin would read in those fonts and add them to the list.

Originally, my thought was to use the PHP-Font-Parser class developed here: https://github.com/Pomax/PHP-Font-Parser but I think if we had a config.json created for each one using fontello.com - we wouldn't need to parse it since the config.json tells us exactly what we want to know.

So basically, we can still have them setup with some basic options such as "fontawesome" "elusive" etc.  But they would be downloaded from fontello versions and stored in their own folders under "fontpacks" or something along those lines...?

I hope I'm making sense with this.
